.github/workflows/actions.yml

```diff
@@ -0,0 +1,29 @@
+on:
```

Comment:
Maybe better file name/action? I'm not sure what each of these new actions is responsible for. This wone looks to build the docfx site and also publish it to _site, but the other one also publishes to docs/v2/_site?

Sorry, just wanting to understand the process :)

Reply:
Deleted that file as it was a failed experiment. Correct workflow file here https://github.com/nzdev/Examine/actions/runs/4149655527/workflow

Reply:
.github/workflows/docfx-gh-pages.yml

Reply:
This is amazing :) The only tweak I think for now is that it's still deploying the docfx site to the root directory (i.e. https://nzdev.github.io/Examine/ instead of https://nzdev.github.io/Examine/v2/). Is that an easy change?

Reply:
Maybe? Let me check

Reply:
It wasn't exactly straight forward. May need to build the existing docs, then the new docs, then move some folders around then push to github pages.

Reply:
Would it be ok to go with just docfx? I'd be happy to help with merging other doc updates pending into docfx

Reply:
I'm no expert at GitHub actions